### PR TITLE
refactor: introduce top level webWallet key

### PR DESCRIPTION
**Pull request by bodymindarts**
Let us start having a top level `<application>:` key to group the app specific settings instead of putting so many settings loosely on top level.

### CODE REVIEW

File: charts/web-wallet/values.yaml [hunk 1]
```diff
@@ -5,18 +5,27 @@ image:
 mobileLayoutImage:
   repository: us.gcr.io/galoy-org/web-wallet-mobile-layout
   digest: "sha256:40e4edbd98cb3d58bda687265fbf0230c6c80745e69778db15218c287ee0e18f"
-useMobileLayout: false
```
Comment (krtk6160) [inline charts/web-wallet/values.yaml:8]: Shouldn't this be kept at top level? This is only used by helm

**Review by krtk6160 — COMMENTED** [review]
One question

File: charts/web-wallet/values.yaml (hunk 1 above)
Comment (bodymindarts) [reply charts/web-wallet/values.yaml:8]: I contemplated that. The thing is I don't like it as a lone top level config (everything else is nested) and I don't know a better key to scope it under.

Comment (krtk6160) [reply charts/web-wallet/values.yaml:8]: Perhaps we can have a `mobileLayout` top level field:

```
mobileLayout:
  enabled: boolean
  image:
    repo: string
    digest: string  
```

**Review by krtk6160 — APPROVED** [review]

**Pull request merged**